logic.js

```diff
@@ -0,0 +1,47 @@
+// Modules
+
+const Tweeter = function () {
+    const _posts = []
+    let postIdCounter = _posts.length
+    let commentIdCounter = 0
+    const getPosts = () => _posts
+    const addPost = (pText) => {
+        _posts.push({ text: pText, id: "p" + (postIdCounter + 1), comments: [] })
+        postIdCounter++
+    }
+    const removePost = (postID) => {
+        for (p of _posts) {
+            if (p.id == postID) {
+                _posts.splice(_posts.indexOf(p), 1)
```

Comment:
In general, you shouldn't use `indexOf` when the values inside the array are objects. This method with only work if the object has the same reference. Instead, use a `for...in` loop and splice at the index you find.